src/lib/http/proxy-agent.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import { SocksProxyAgent } from 'socks-proxy-agent';
+import { HttpsProxyAgent } from 'https-proxy-agent';
+import { HttpProxyAgent } from 'http-proxy-agent';
+import { getProxyForUrl } from 'proxy-from-env';
+import debug from 'debug';
```

Comment:
@frank-cerny while testing I noticed that the debug line doesn't work because the first argument is the environment part.
This change should fix it. 

```suggestion
import debugLib from 'debug';
const debug = debugLib( 'vip:proxy-agent' );
```